### PR TITLE
Switch to LS AAI dcp-714

### PR DESCRIPTION
**Pull request by jacobwindsor**
dcp-714

image

### CODE REVIEW

**Comment by prabh-t** [comment]
Nice! :) maybe we can drop the RI ?

**Review by prabh-t — DISMISSED** [review]

**Comment by jacobwindsor** [comment]
@prabh-t not sure. I think that is the name. Well the full name is European life sciences research infrastructures aha

File: src/app/login/login.component.html [hunk 1]
```diff
@@ -1,23 +1,20 @@
 <div class="vf-intro">
   <h1 class="vf-intro__heading vf-intro__heading--has-tag">Log in <a class="vf-badge vf-badge--primary vf-badge--phases">beta</a></h1>
-  <p class="vf-lede">To submit your single-cell sequencing data to the Human Cell Atlas, you will need an Elixir account.</p>
+  <p class="vf-lede">To submit your single-cell sequencing data to the Human Cell Atlas, you will need a LifeScience RI account.</p>
```
Comment (amnonkhen) [inline src/app/login/login.component.html:3]: Should we link to the LS website?

File: src/app/login/login.component.html [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
-          Use your existing Elixir account to sign in. If you do not have an Elixir account, you can associate an existing
-          email account with Elixir and return to sign in here.
+          Use your existing LifeScience RI (previously Elixir) account to sign in. If you do not have an account, you can associate an existing
+          email account with LifeScience RI and return to sign in here.
         </p>
         <br/>
         <p>
           <a (click)="login()" href="javascript:void(0)" class="vf-link" style="margin:0 50px 0 0">
-            <img src="assets/login-button-blue-small.png" alt="[ELIXIR Login]">
+            <img src="assets/ls-login.png" alt="[LS AAI Login]" width="220">
```
Comment (amnonkhen) [inline src/app/login/login.component.html:14]: Well done with prefix based naming!

**Review by amnonkhen — COMMENTED** [review]
LGTM.
Do they have svgs or just pngs?

**Review by amnonkhen — DISMISSED** [review]

**Comment by jacobwindsor** [comment]
@amnonkhen just pngs

**Comment by jacobwindsor** [comment]
Merging without re-approvals since only a tiny change to drop RI occurred

**Pull request merged**